src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -1218,4 +1285,22 @@ private List<AttachmentInfo> getAttachmentInfoForParameter(final Parameter param
             .secureToken(value.getSecureToken()).build())
         .collect(Collectors.toList());
   }
+
+  /**
+   * Calls RPA to Open Envelope
+   *
+   * @param userEmail
+   * @param password
+   * @param externalReferenceId
+   * @return
+   */
+  public void  callOpenEnvelopeAndUpdateSupplier(final String userEmail, final String password,
```

Comment:
@SrinivasaDontiboyina Can you make this as a generic method because we are using same method in supplierService as well.

Reply:
Renamed the method name as I am not updating supplier using only one operation ,
 in your case your doing 2 operations (opening envelope and updating score). 
 may not be worthy creating a method for this